agent/ecscni/plugin.go

```diff
@@ -125,6 +125,15 @@ func (client *cniClient) setupNS(cfg *Config) (*current.Result, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "cni setup: invoke bridge plugin failed")
 	}
+	if cfg.AppMeshCNIEnabled {
+		// Invoke app mesh plugin ADD command
+		seelog.Debug("[APPMESH] Starting appmesh setup done")
+		result, err = client.add(runtimeConfig, cfg, client.createAppMeshConfig)
```

Comment:
how about change to result, err := client .... So we only keep the result and err within the if block, since it only matters within the block.

Reply:
Can you confirm appmesh plugin will return a non nil result? It seems during our testing appmesh plugin result caused agent to exit 2